src/config.c

```diff
@@ -79,18 +79,20 @@ CONFIG_GETTER(getExtLoad) {
 
 // SAFEMODE
 CONFIG_SETTER(setSafemode) {
-  config->concurrentMode = 0;
+  RedisModule_Log(RSDummyContext, REDISMODULE_LOGLEVEL_WARNING, "SAFEMODE option is deprecated, and has no effect");
   return REDISMODULE_OK;
 }
 
-CONFIG_BOOLEAN_GETTER(getSafemode, concurrentMode, 1)
+// dummy getter for safemode. Assuming that `maxDocTableSize` is never 0, we always return "true"
+CONFIG_BOOLEAN_GETTER(getSafemode, maxDocTableSize, 0)
```

Comment:
I do not like this hack with `maxDocTableSize`, lets create another implementation that returns const 1.